i18n/ui/lbm-ui_en-US.po

```diff
@@ -102,6 +102,14 @@ msgstr "You Purchased"
 msgid "Your coins are on their way to:"
 msgstr "Your coins are on their way to:"
 
+#: On screen: completed
+msgid "Print receipt"
+msgstr "Print receipt"
+
+#: On screen: completed
+msgid ""
+msgstr ""
+
```

Comment:
Hmm, looks like something useless has a i18n tag

Reply:
Oops, you're right